pycbc/workflow/datafind.py

```diff
@@ -977,6 +949,14 @@ def run_datafind_instance(cp, outputDir, connection, observatory, frameType,
     if tags is None:
         tags = []
 
+    # Determine if we should override the default datafind server
+    if cp.has_option_tags("workflow-datafind",
+                          "datafind-ligo-datafind-server", tags):
+        datafind_server = cp.get_opt_tags("workflow-datafind",
+                                        "datafind-ligo-datafind-server", tags)
```

Comment:
```suggestion
        datafind_server = cp.get_opt_tags(
            "workflow-datafind",
            "datafind-ligo-datafind-server",
            tags)
```

Reply:
If doing this, the `)` should be on the following line to follow the convention set out in `black` with complies with PEP-8. I couldn't modify the suggestion to add that.

Reply:
Updated now